### ISSUE TITLE
[DevOps] Proof-of-Concept for CloudWatch Metric Streams to Datadog

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-team#83403]
## Description
See [this Slack thread](https://dsva.slack.com/archives/C02G6AB3ZRS/p1715963957376419).

**TL;DR**: The pushback to using service accounts is very significant and I don't know if we'll be able to surmount it. Therefore, we should investigate pushing metrics from AWS to Datadog rather than pulling them.

[CloudWatch Metric Streams](https://www.datadoghq.com/blog/amazon-cloudwatch-metric-streams-datadog/) is an appealing possibility, but we have no experience with it and therefore can't easily assess it.

We should figure out a PoC to determine whether it can meet our needs.

## Tasks
- [x] [DevOps] Discovery on CloudWatch Metric Streams and Firehose
- [x] [DevOps] Determine lift for Terraform for e.g. some DynamoDB metrics that are currently pulled from Tevi Datadog.
- [x] [DevOps] Implement for `dev`.
- [ ] [DevOps] Run some tests or perform operations that can reflect some changes in Datadog.
- [ ] [DevOps] Determine whether or not to move forward with this more broadly.

## Acceptance Criteria
- [ ] CloudWatch Metric Streams have been evaluated as a way of pushing data to Datadog rather than pulling from Datadog.
- [ ] Findings documented in this thread.
- [ ] Followup ticket(s) opened as necessary.

**Comment by ndouglas** [comment]
I had trouble with this; I was able to configure an IAM Role with an assume-role policy that does the equivalent, though, with less maintenance on our end.
